R/AbstractPackageReporter.R

```diff
@@ -5,7 +5,7 @@
 #'              The class is not meant to be instantiated, but inherited from and its methods
 #'              overloaded such that each Metric implements certain functionality.
 #' @importFrom R6 R6Class
-#' @section Public Methods:
+#' @section Public:
```

Comment:
point for later, I guess, but we need to standardize on how we document R6. I like having a division between `Public Methods` and `Public Members`, personally (instead of one big `Public` section). But def willing to be convinced